client/Log.h

```diff
@@ -1,4 +1,12 @@
+#pragma once
+
 #include <string>
+#include <cstdint>
+
+#ifndef _WINDEF_
+class HINSTANCE__; // Forward or never
+typedef HINSTANCE__* HINSTANCE;
+#endif

```

Comment:
I was split between placing `std::string FormatLoadAddress(HINSTANCE value)` in Log.h or in main.cpp. If moved to main.cpp, this forward declare can be removed.

Reply:
Hmm, a bit of a toss up. The problem with the forward declare, is it depends on effectively private details of the Windows header. It would be safer to just include the Windows header. That adds a bit to compile time, though is probably the most correct thing to do.

I'll review in detail in the morning.